lambdas/id_sync/src/pds_details.py

```diff
@@ -31,26 +29,12 @@ def pds_get_patient_details(nhs_number: str) -> dict:
         patient = pds_service.get_patient_details(nhs_number)
         return patient
     except Exception as e:
-        msg = f"Error getting PDS patient details for {nhs_number}"
+        msg = "Error retrieving patient details from PDS"
         logger.exception(msg)
         raise IdSyncException(message=msg, exception=e)
```

Comment:
Don't see why we have `inner_exception` in `IdSyncException`. This could just be `raise IdSyncException(message=msg) from e`